components/admin/lessons/AdminLessonNav.tsx

```diff
@@ -1,18 +1,22 @@
-import React from 'react'
+import React, { FunctionComponent } from 'react'
 import styles from '../../../scss/adminLessonNav.module.scss'
 
 type Props = {
   tabs: { tabName: string; urlPageName: string }[]
```

Comment:
Looking at the current state of this component, I'm noticing that `tabName` and `urlPageName` is not used for any logic within this component, so could we just have an object that could be anything?